docs/deploy-with-docker.md

```diff
@@ -19,7 +19,9 @@
 
 # Building and Launching with Docker
 
-To build a Dashboard with Docker, you simply download the `Dockerfile` file from the **root directory** to your device (no need to download all source codes) then follow this guide.
+**NOTE:** We support official Docker Image, please visit [DockerHub](https://hub.docker.com/r/apache/apisix-dashboard) for more information. The following steps are for building Docker Image manually.
```

Comment:
docker image cannot claim to be an official image.

Ref to APISIX.

![image](https://user-images.githubusercontent.com/4413028/104264006-b3fe2580-54c5-11eb-906e-181be6148ab2.png)

Reply:
fine